.github/workflows/ci.yml

```diff
@@ -105,20 +108,23 @@ jobs:
       - name: Set wait for image
         id: wait-for-image
         run: |
-          if [[ ${GITHUB_REGISTRY_WAIT_FOR_IMAGE} == "true" ]]; then
+          if [[ ${GITHUB_REGISTRY_WAIT_FOR_IMAGE} == 'true' ]]; then
               echo "::set-output name=wait-for-image::true"
           else
               echo "::set-output name=wait-for-image::false"
           fi
       - name: Selective tests
         id: selective-tests
-        run: ./scripts/ci/selective_tests.sh
+        run: ./scripts/ci/selective_tests.sh ${{github.sha}}
 
   helm-tests:
     timeout-minutes: 5
     name: "Checks: Helm tests"
     runs-on: ubuntu-latest
-    if: github.repository == 'apache/airflow' || github.event_name != 'schedule'
+    needs: [build-info]
+    if: >
+      needs.build-info.outputs.needs-helm-tests == 'true' &&
+      (github.repository == 'apache/airflow' || github.event_name != 'schedule')
```

Comment:
(Not a change on this PR, but why do we care what repository we are for helm chart tests?

Reply:
We do it for all the repos and only for scheduled workflows (hence the || schedule). Since we are using the CI builds nightly, if someone forks our repository, those scheduled builds did (read further) executed also in forks. And several people pointed out that it is rather intrusive thing - to get people use their free jobs from GitHub actions on the forks. Public repos are free, but individuals have limits on a number of jobs and API calls per hour and running scheduled actions on someone else's fork is a bad idea. 

That's why the recommendation (in many threads) was to only run scheduled workflows in the main repository doing this kind of if statement (it basically means the the workflow will run in you fork for pushes and PRs but not for schedules).

This is going to change soon however. It has not been publicly announced by GitHub on their web pages (also the fact that scheduled workflows run in forks was not an obvious thing) but I recently - 1st of October got an email stating that the policy, and we will be able to remove this if on October 27, 2020:

> Updates to GitHub Actions usage policy: scheduled workflows in inactive and forked public repositories.

> Scheduled workflows will be disabled by default in forks of public repos and in public repos with no activity for 60 consecutive days.
We’re making two changes to the usage policy for GitHub Actions. These changes will enable GitHub Actions to scale with the incredible adoption we’ve seen from the GitHub community. Here’s a quick overview:

> Starting today, scheduled workflows will be disabled by default in new forks of public repositories.
Scheduled workflows will be disabled in public repos with no activity for 60 consecutive days.
We’re dedicated to helping the open source community be productive, which is why GitHub Actions is free for the 50 million developers who work with public repositories. These changes will help us ensure a great GitHub Actions experience for the next 50 million developers on GitHub.

> To further evaluate how this will affect you, please read our FAQ below and contact GitHub Support if you run into any problems or need help. You can also always get up to date information about our usage policies in our Docs.

> Thank you for using GitHub Actions 💙

> What will happen to scheduled workflows in forks I already have?
> If you already have scheduled workflows in forks of public repositories, they will be disabled on October 27, 2020. Repository owners will receive a reminder email 7 days prior and can choose to keep the workflows running at that time if they are needed.

> I have inactive public repos, are my scheduled workflows being disabled?
We are not evaluating public repos for 60 consecutive days of inactivity until October 27, 2020. After that, public repos with 60 consecutive days of inactivity will have scheduled workflows disabled. Inactive repo owners will receive a reminder email 7 days before any workflows are disabled and can choose to keep them running at that time.

> Can I re-enable disabled workflows?
Yes! You can easily re-enable any workflow that has been disabled by using the brand new feature to enable & disable workflows. You can enable a disabled workflow with a few clicks.